composer.json

```diff
@@ -23,17 +23,17 @@
     "require": {
         "php": "^7.1",
         "jeremykendall/php-domain-parser": "^5.4",
-        "illuminate/support": "^5.5",
-        "illuminate/config": "^5.5"
+        "illuminate/support": "^6.0",
```

Comment:
why can't you just do "^5.5|^6.0" ? and do the same with the other dependencies 🤔 
I see no point in dropping support for older version since the package code is not even changed.

Apart from that if the tests continue to stay green I'm 👍 with your PR.

Reply:
Sure! I'll take a look at supporting both later this week, thanks for your reply.